linux_os/guide/system/network/network-ipv6/disabling_ipv6/sysctl_net_ipv6_conf_default_disable_ipv6/rule.yml

```diff
@@ -44,7 +44,11 @@ ocil: |-
     files in <tt>/etc/sysctl.d</tt>:
     <pre>$ grep -r ipv6 /etc/sysctl.d</pre>
 
+{{% if product in ["sle12", "sle15", "slmicro5"] %}}
+platform: ipv6[enabled]
```

Comment:
@teacup-on-rockingchair  Sorry for noticing it late, but I suspect this rule suffers from the same problem, I can't find it in the built slmicro5 data stream.